geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/RemoveCommand.java

```diff
@@ -90,7 +90,14 @@ public ResultModel remove(
     }
 
     dataResult.setKeyClass(keyClass);
+    ResultModel result = null;
```

Comment:
Setting the value of `result` here is redundant, since it will always be assigned in the if/else block below.

Reply:
I've removed the redundant initialization.